### PR TITLE
Refactoring - retrieve user ID

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Refactoring - retrieve user ID
(it will be used in several parts of code)

## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: ols/app/endpoints/ols.py [hunk 1]
```diff
@@ -97,6 +96,12 @@ def conversation_request_debug_api(llm_request: LLMRequest) -> LLMResponse:
     )
 
 
+def retrieve_user_id(auth: Any = Depends(auth_dependency)) -> str:
```
Comment (onmete) [inline ols/app/endpoints/ols.py:99]: Is the auth type correct?

Comment (tisnik) [reply ols/app/endpoints/ols.py:99]: it's inherited from callee

Comment (tisnik) [reply ols/app/endpoints/ols.py]: updated @onmete

**Pull request merged**